src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -20,7 +20,7 @@
 ## Upcoming Release
 * Added support for custom RG with suffix while creating or modifying policy for workload type AzureVM.
 * Added TLaD warning https://aka.ms/TLaD.
```

Comment:
```suggestion
```
You don't need this entry

Reply:
This is added for change related to backup warning message.Should I remove it or club the newly added change log with site recovery and backup

Reply:
Yes, please.

Reply:
I have updated. Please check now.